### PR TITLE
Adding Protobuf support for truck

### PR DESCRIPTION
**Pull request by sasathy**
# Description and Motivation 

...

# Change type 

- [ ] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [ ] All tests passed
- [ ] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

---
This change is [Reviewable](https://reviewable.io/reviews/azure/device-simulation-dotnet/235)

### CODE REVIEW

File: docs/API_SPECS_DEVICE_MODELS.md [hunk 1]
```diff
@@ -485,4 +485,4 @@ DELETE /v1/devicemodels/${id}
 Response:
 ```
 200 OK
-```
\ No newline at end of file
+```
```
Comment (hathind-ms) [inline docs/API_SPECS_DEVICE_MODELS.md]: what is the change here? if nothing has changed then revert to master version for this file to keep history clean

File: docs/API_SPECS_SIMULATIONS.md [hunk 2]
```diff
@@ -617,4 +617,4 @@ Simulations can be deleted using the DELETE method.
 
 ```
 DELETE /v1/simulations/1
-```
\ No newline at end of file
+```
```
Comment (hathind-ms) [inline docs/API_SPECS_SIMULATIONS.md]: same as above: what is the change here? if nothing has changed then revert to master version for this file to keep history clean

Comment (sasathy) [reply docs/API_SPECS_SIMULATIONS.md]: done

File: docs/API_SPECS_DEVICE_MODELS.md (hunk 1 above)
Comment (sasathy) [reply docs/API_SPECS_DEVICE_MODELS.md]: done

**Pull request merged**